File: Programação Orientada a Objetos/Workspace_POO/Java_POO/Java_POO/src/Arrays/Testing_bmiCalc.java
package Arrays;

public class Testing_bmiCalc {
	public static void main(String[] args) {
		// Weight
		double weight = 70;
		// Height
		double height = 1.7;
		// Code
		double bmi = weight / Math.pow(height, 2);
		// Diagnostic
		String diagnostic = (bmi > 20 && bmi <= 25) ? "Ideal weight" : "Not ideal weight";
		System.out.printf("IMC: %.2f\nDiagnostic: %s\n", bmi, diagnostic);
	}
}

File: POO/Workspace/CS/Arrays/Arrays/Arrays/Arrays/Testing_bmiCalc.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Arrays
{
    class Testing_bmiCalc
    {
        public Testing_bmiCalc()
        {
            // Weight
            double weight = 70;
            // Height
            double height = 1.7;
            // Code
            double bmi = weight / Math.Pow(height, 2);
            // Diagnostic
            String diagnostic = (bmi > 20 && bmi <= 25) ? "Ideal weight" : "Not ideal weight";
            Console.WriteLine("IMC: {0:F2}\nDiagnostic: {1}\n", bmi, diagnostic);
        }
    }
}
